src/commands/distribute/release.ts

```diff
@@ -7,6 +7,12 @@ import * as Request from "request";
 import * as Path from "path";
 import * as Pfs from "../../util/misc/promisfied-fs";
 import { DefaultApp } from "../../util/profile";
+import FileUploadClient, { IFileUploadClientSettings, IProgress, IUploadResults, MessageLevel } from "appcenter-file-upload-client";
```

Comment:
What is this syntax with one name outside the braces and others inside? What does this do?

Reply:
FileUploadClient is the default export class, whereas the others are not. Here it is its definition: https://github.com/Microsoft/appcenter-file-upload-client/blob/master/src/index.ts